tools/depends/native/TexturePacker/src/decoder/GifHelper.cpp

```diff
@@ -43,14 +43,7 @@ int ReadFromVfs(GifFileType* gif, GifByteType* gifbyte, int len)
   return gifFile->Read(gifbyte, len);
 }
 
-GifHelper::GifHelper() :
-  m_imageSize(0),
-  m_pitch(0),
-  m_loops(0),
-  m_numFrames(0),
-  m_filename(""),
-  m_gif(nullptr),
-  m_pTemplate(nullptr)
+GifHelper::GifHelper() : m_filename("")
```

Comment:
`m_filename`is a `std::string` and its default ctor contructs an empty string. So, no need to call the ctor with `""`here.

Reply:
I know, but it was there before, removed it now manually.